docs/spring-boot/app-starter.md

```diff
@@ -124,138 +124,4 @@ public class ExtAppSpringApplication {
-        200: 
-          description: Success
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/AppToken'
-        400:
-          description: Bad Request
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        401:
-          description: Unauthorized
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-  '/bdk/v1/app/tokens':
-    post:
-      tags:
-      - Circle Of Trust
-      summary: Validate Tokens
-      operationId: validateTokens
-      requestBody:
-        content:
-          application/json:
-            schema: 
-              $ref: '#/components/schemas/TokenPair'
-      responses:
-        204: 
-          description: Success
-          content: {}
-        400:
-          description: Bad Request
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        401:
-          description: Unauthorized
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
-  '/bdk/v1/app/jwt':
-    post:
-      tags:
-      - Circle Of Trust
-      summary: Validate JWT
-      operationId: validateJwt
-      requestBody:
-        content:
-          application/json:
-            schema: 
-              $ref: '#/components/schemas/Jwt'
-      responses:
-        200: 
-          description: Success
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/UserInfo'
-        400:
-          description: Bad Request
-          content: 
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-        401:
-          description: Unauthorized
-          content: 
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-components:
-  schemas:
-    AppToken:
-      type: object
-      properties:
-        appToken:
-          type: string
-    TokenPair:
-      type: object
-      properties:
-        appToken:
-          type: string
-        symphonyToken:
-          type: string
-    Jwt:
-      type: object
-      properties:
-        jwt:
-          type: string
-    UserInfo:
-      type: object
-      properties:
-        userId:
-          type: number
-          format: int64
-    Error:
-      type: object
-      properties:
-        status:
-          type: number
-          format: int32
-        errorCode:
-          type: string
-        message:
-          type: array
-          items: 
-            type: string
-```
+[Circle of Trust API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/SymphonyPlatformSolutions/symphony-api-client-java/master/docs/spring-boot/circle-of-trust.yaml)
```

Comment:
[Looks nice already](https://editor.swagger.io/?url=https://raw.githubusercontent.com/SymphonyPlatformSolutions/symphony-api-client-java/60e7ac9f14eebc8df644f8f7290343e8c8215dec/docs/spring-boot/circle-of-trust.yaml) :)